experimental/conventional-model/terraform/modules/ops/services.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  services = var.enable_apis ? [
```

Comment:
I like the simplification of a list of services to be enabled, but why gate this behind a var?

Reply:
gated behind a var as suggested by best practices.  defaulted to true but gives the option to relieve Terraform from managing services.